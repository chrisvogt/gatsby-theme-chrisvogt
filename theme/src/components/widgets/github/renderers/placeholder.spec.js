import React from 'react'
import Placeholder from './placeholder'
import renderer from 'react-test-renderer'

describe('Placeholder', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Placeholder />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
