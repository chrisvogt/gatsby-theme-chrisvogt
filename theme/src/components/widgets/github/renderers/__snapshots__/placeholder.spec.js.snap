// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHub Placeholder Renderer matches the snapshot 1`] = `
<div
  className="show-loading-animation"
>
  <div
    className="css-vd02hx-Flex"
  >
    <div>
      <div
        className="rect-shape"
        style={
          {
            "backgroundColor": "#efefef",
            "height": 40,
            "marginBottom": "2em",
            "marginRight": 10,
            "width": 40,
          }
        }
      />
    </div>
    <div
      className="css-fjakwv"
    >
      <div
        className="text-block"
        style={
          {
            "width": "100%",
          }
        }
      >
        <div
          className="text-row"
          style={
            {
              "backgroundColor": "#efefef",
              "height": "1em",
              "marginTop": 0,
              "maxHeight": "33.333333333333336%",
              "width": "97%",
            }
          }
        />
        <div
          className="text-row"
          style={
            {
              "backgroundColor": "#efefef",
              "height": "1em",
              "marginTop": "0.7em",
              "maxHeight": "33.333333333333336%",
              "width": "100%",
            }
          }
        />
      </div>
    </div>
  </div>
  <div
    className="text-block"
    style={
      {
        "width": "100%",
      }
    }
  >
    <div
      className="text-row"
      style={
        {
          "backgroundColor": "#efefef",
          "height": "1em",
          "marginTop": 0,
          "maxHeight": "20%",
          "width": "97%",
        }
      }
    />
    <div
      className="text-row"
      style={
        {
          "backgroundColor": "#efefef",
          "height": "1em",
          "marginTop": "0.7em",
          "maxHeight": "20%",
          "width": "100%",
        }
      }
    />
    <div
      className="text-row"
      style={
        {
          "backgroundColor": "#efefef",
          "height": "1em",
          "marginTop": "0.7em",
          "maxHeight": "20%",
          "width": "94%",
        }
      }
    />
  </div>
</div>
`;
